server/neptune/workflows/internal/prrevision/workflow.go

```diff
@@ -17,6 +17,9 @@ import (
 )
 
 const (
+	TaskQueue       = "pr_revision"
+	OpenPullRequest = "open"
```

Comment:
how bout 
```
type PullRequestState string
```

and then just create constants from that, otherwise we'll get carried away and just use raw strings everywhere